### PR TITLE
#530, #529, #501, #494, #458 - Initial commit

### PR DESCRIPTION
**Pull request by neilotte**

### CODE REVIEW

File: src/cco-merged/CommonCoreOntologiesMerged.ttl
Comment (johnbeve) [inline src/cco-merged/CommonCoreOntologiesMerged.ttl:1]: ###  http://www.ontologyrepository.com/CommonCoreOntologies/example_of_usage

 Should be removed from the merged file (don't see it in any module though FYI)

Comment (neilotte) [reply src/cco-merged/CommonCoreOntologiesMerged.ttl:1]: @johnbeve Good catch. I think this file is regenerated anyway. I'll do that in next commit.

Comment (johnbeve) [reply src/cco-merged/CommonCoreOntologiesMerged.ttl:1]: @neilotte Didn't realize you could see, I've been updating this comment so there might be a bit more to check

Comment (neilotte) [reply src/cco-merged/CommonCoreOntologiesMerged.ttl:1]: @johnbeve I've updated the merged file, but I believe the dev team is automating generation of the modal file so I won't touch it further here (@mark-jensen  or @cameronmore are welcome to though, here or on a subsequent MR).

Comment (neilotte) [reply src/cco-merged/CommonCoreOntologiesMerged.ttl:1]: @mark-jensen @johnbeve Updated two sparql queries that used cco:definition as well.

**Review by mark-jensen — APPROVED** [review]

**Pull request merged**